crates/toml_datetime/src/ser.rs

```diff
@@ -50,14 +50,14 @@ impl DatetimeSerializer {
         Self { value: None }
     }
 
-    /// See [`serde::ser::SerializeStruct::serialize_field`]
+    /// See [`serde_core::ser::SerializeStruct::serialize_field`]
```

Comment:
Is this okay? Linking to `serde` instead would require that to be a dependency. I can add it as an optional behind a `__private_docs` feature or somesuch if you want (that's what we do in axum to link to things that aren't [direct] dependencies).

Reply:
Wouldn't having `serde` as a dev dependency work?